packages/auth-server/src/account/repository.js

```diff
@@ -0,0 +1,91 @@
+import { Prisma } from '@prisma/client';
```

Comment:
this has come from appeals api, can potentially remove some of it once in/move all user logic to this api